.github/workflows/build.yml

```diff
@@ -47,6 +47,10 @@ jobs:
         run: |
           . /home/firedrake/firedrake/bin/activate
           python $(which firedrake-clean)
+          cd run_scripts_testing/ice_shelf_cavity/isomip_plus/
+          # Run 3d isomip plus for 5 time steps
+          make
+          cd ../../..
```

Comment:
```suggestion
          # Run 3d isomip plus for 5 time steps
          make -C run_scripts_testing/ice_shelf_cavity/isomip_plus/
```

Reply:
Fine for now, but if we're going to do this for multiple run scripts we probably want some system in place. This is currently a little fragile, see e.g. the issues with the log below...